influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
`SELECT value, mean(value)` isn't a valid query. If you have an aggregate, you have a group by time, so it doesn't work with that.

The exception to this would be `SELECT host, min(value) from cpu`, which will be a valid query when it's wired up (#1577). However, that will still be an aggregate query, not a raw one.